src/uct/ib/rc/base/rc_ep.c

```diff
@@ -176,12 +178,24 @@ void uct_rc_ep_get_bcopy_handler_no_completion(uct_rc_iface_send_op_t *op,
     uct_rc_iface_send_desc_t *desc = ucs_derived_of(op, uct_rc_iface_send_desc_t);
 
     VALGRIND_MAKE_MEM_DEFINED(resp, desc->super.length);
+    ++desc->super.iface->tx.gets_available;
 
     desc->unpack_cb(desc->super.unpack_arg, resp, desc->super.length);
 
     ucs_mpool_put(desc);
 }
 
+void uct_rc_ep_get_zcopy_completion_handler(uct_rc_iface_send_op_t *op,
+                                            const void *resp)
+{
+    ++op->iface->tx.gets_available;
+
+    /* Revert op handler to the general one, suitable for non-get ops */
+    op->handler = uct_rc_ep_send_op_completion_handler;
```

Comment:
why do we need to restore if uct_rc_txqp_add_send_comp() is doing `op->handler   = handler`

Reply:
`uct_rc_txqp_add_flush_comp` does not set it.

Reply:
so better add to flush_comp as well